app/kitchensink/index.html

```diff
@@ -1509,6 +1509,25 @@ <h2 id="Form Controls" class="page-header">Form Controls
                       <label for="t7" data-on="ON" data-off="OFF">Disabled off</label>
                     </div>
                   </div>
+
+                  <div class="row">
+                    <div class="col-md-6">
+                    <p>Use the toggle switches, instead of radio buttons, for two or more specific options. </p>              
+
+                    <fieldset >
+                      <legend class="sr-only">Select options</legend>
+                      <div class="switch-toggle">
+                        <input id="option1" name="selectOption" type="radio" checked>
+                        <label for="option1" onclick="">Option 1</label>
+                        <input id="option2" name="selectOption" type="radio">
+                        <label for="option2" onclick="">Option 2</label>
+                        <input id="option3" name="selectOption" type="radio">
+                        <label for="option3" onclick="">Option 3</label>
+                        <a class="btn btn-info"></a>
```

Comment:
the highlight color is governed by what class is placed on the a tag? So, if I put in btn-success or btn-warning it will be changed?

Reply:
Yes, it works.

Reply:
Sure, is that something we want to control in another way? say class="highlight-color"

Reply:
It pulls in bootstrap classes, no not needed now, but if we move to MDL then we need to change the class structure.